docs/en/cookbook/entities-in-session.rst

```diff
@@ -29,40 +31,20 @@ the session into a managed Doctrine object looks like this:
     $em = GetEntityManager(); // creates an EntityManager 
 
     session_start();
-    if (isset($_SESSION['user']) && $_SESSION['user'] instanceof User) {
-        $user = $_SESSION['user']; 
-        $user = $em->merge($user);
+    if (isset($_SESSION['user'])) {
+        $user = $em->find(User::class, $_SESSION['user']);
```

Comment:
Maybe, but it's not a direct replacement. It could technically allow weirdnesses like _sessions for deleted users_.